content/docs/refs-and-the-dom.md

```diff
@@ -165,25 +209,27 @@ function CustomTextInput(props) {
 }
 
 class Parent extends React.Component {
+  constructor(props) {
+    super(props);
+    this.inputElement = React.createRef();
+  }
   render() {
     return (
-      <CustomTextInput
-        inputRef={el => this.inputElement = el}
-      />
+      <CustomTextInput inputRef={this.inputElement} />
     );
   }
 }
 ```
 
-In the example above, `Parent` passes its ref callback as an `inputRef` prop to the `CustomTextInput`, and the `CustomTextInput` passes the same function as a special `ref` attribute to the `<input>`. As a result, `this.inputElement` in `Parent` will be set to the DOM node corresponding to the `<input>` element in the `CustomTextInput`.
+In the example above, `Parent` passes its class property `this.inputElement` as an `inputRef` prop to the `CustomTextInput`, and the `CustomTextInput` passes the same ref as a special `ref` attribute to the `<input>`. As a result, `this.inputElement.value` in `Parent` will be set to the DOM node corresponding to the `<input>` element in the `CustomTextInput`.
```

Comment:
nit: I think this example might read clearer if the parent class property was `inputRef` as well.